textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -32,78 +32,86 @@
 import edu.uci.ics.textdb.common.field.TextField;
 
 public class Utils {
-    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException{
+    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException {

```

Comment:
It will be greatly appreciated if you can move the reformatting changes to a different PR.  I hope that's not hard. That can make my life much easier :-)